ttnn/cpp/pybind11/__init__.cpp

```diff
@@ -11,13 +11,17 @@
 #include "types.hpp"
 #include "reports.hpp"
 
+#include "ttnn/experimental/tt_lib/csrc/tt_lib_bindings.hpp"
 #include "operations/__init__.hpp"
 
 namespace py = pybind11;
 
 PYBIND11_MODULE(_ttnn, module) {
     module.doc() = "Python bindings for TTNN";
 
+    auto m_deprecated = module.def_submodule("deprecated", "tt_lib deprecated bindigns tensor, device, profiler..");
```

Comment:
spelling

Reply:
pedantic because it seems like an important import for ppl

Reply:
Thank you!